charts/velero/README.md

```diff
@@ -6,11 +6,11 @@ Velero has two main components: a CLI, and a server-side Kubernetes deployment.
 
 ## Installing the Velero CLI
 
-See the different options for installing the [Velero CLI](https://velero.io/docs/v1.2.0/install-overview/#install-the-cli).
+See the different options for installing the [Velero CLI](https://velero.io/docs/v1.3.0/install-overview/#install-the-cli).
```

Comment:
This link is no longer valid due to some docs reorg, needs to be `https://velero.io/docs/v1.3.0/basic-install/#install-the-cli`